packages/client/src/orders/types/getShipmentTrackings.types.ts

```diff
@@ -1,7 +1,7 @@
 import type { Config } from '../..';
 import type { Trackings } from './tracking.types';
 
-export type GetTrackings = (
+export type GetShipmentTrackings = (
   trackingCodes: string,
   config?: Config,
 ) => Promise<Trackings>;
```

Comment:
To make it more distinguishable from `Tracking` in analytics, what do you think of renaming the `Trackings` type to `ShipmentTrackings` and `Tracking` to `ShipmentTracking` as well?

Reply:
Agreed. Changed